frontend/src/static/js/v_authorship.js

```diff
@@ -277,6 +291,16 @@ window.vAuthorship = {
       this.updateSelectedFiles();
     },
 
+    isValidFile(file) {
+      if (this.info.isMergeGroup) {
+        const fileAuthorCnt = Object.entries(file.authorContributionMap)
+            .filter((authorCount) => !this.isUnknownAuthor(authorCount[0]))
```

Comment:
can we use some() here instead of computing the length?

Reply:
Sure, no problem